lib/ar2dto/model_config.rb

```diff
@@ -8,12 +8,12 @@ def initialize(model)
       @model = model
     end
 
-    def setup_config(configs)
-      @configs = AR2DTO::Config.instance.as_json.merge(configs.as_json)
+    def setup_configs(configs)
+      @configs ||= configs
     end
 
     def except
-      @except ||= configs["except"]
+      @except ||= Array(AR2DTO::Config.instance.except) | Array(configs[:except])
```

Comment:
`|` or `||`

Reply:
`Array` `|` is the set union operator https://ruby-doc.org/core-2.7.0/Array.html#method-i-7C